tests/config_test.py

```diff
@@ -23,6 +26,20 @@ def test_load_config_too_permissive(file_config):
     )
 
 
+def test_load_config_bad_output_dir(file_config):
+    with open(file_config.cfg) as stream:
+        contents = json.load(stream)
+    contents['output_dir'] = os.getcwd()
```

Comment:
a test failure now blows away the working directory, this is not a good idea

Reply:
Not sure how - are you talking about the cleanup of the tmpdir fixture that the file_config fixture used?  This just modified file contents of a file within the tmpdir.

Reply:
comment out your fix for example